pkg/codegen/convert/plugin_mapper.go

```diff
@@ -121,7 +121,7 @@ type pluginMapper struct {
 func NewPluginMapper(ws Workspace,
 	providerFactory ProviderFactory,
 	key string, mappings []string,
-	installProvider func(tokens.Package) *semver.Version,
+	installPulumiProvider func(tokens.Package) *semver.Version,
```

Comment:
Why are we renaming this? It's only _ever_ going to be a pulumi provider.

Reply:
that is true but I wanted to make it explicit (especially when I was going to warn to let a user/us know an SDK was being generated for debugging purposes) that this was a pulumi provider and does not handle TF bridging.